beacon-chain/forkchoice/interfaces.go

```diff
@@ -63,10 +60,11 @@ type Getter interface {
 	BestJustifiedCheckpoint() *forkchoicetypes.Checkpoint
 	NodeCount() int
 	HighestReceivedBlockSlot() primitives.Slot
-	HighestReceivedBlockRoot() [32]byte
```

Comment:
Is this not used either?

Reply:
Indeed not used. I can also remove the implementation